packages/neo4j-driver/test/temporal-types.test.js

```diff
@@ -1439,7 +1439,24 @@ describe('#integration temporal-types', () => {
       expect(records.length).toEqual(1)
 
       const value = records[0].get(0)
-      expect(value).toEqual(expectedValue)
+
+      if (
+        expectedValue.timeZoneId != null &&
+        value.timeZoneOffsetSeconds != null &&
+        neo4j.isDateTime(value) &&
+        neo4j.isDateTime(expectedValue)) {
+        expect(value).toEqual(jasmine.objectContaining({
+          year: expectedValue.year,
```

Comment:
Maybe add a comment here. I'm 100% sure about the intention of this special treatment. (same for the change further down in this file)

Reply:
The DateTime with zone are created without the offset, but they return with the offset from the utc patch servers. This changes have the intention of ignoring the offset in this comparison.

Reply:
What if it was 
```
        expectedValue.timeZoneId != null &&
        expectedValue.timeZoneOffsetSeconds != null &&
```
? Would that do the same thing? If so, I think it feels cleaner to know in advance what the result should be and not decide how to check the result depending on what it looks like (whenever possible, ofc.)

Reply:
```
expectedValue.timeZoneId != null &&
        expectedValue.timeZoneOffsetSeconds != null &&
```

There is not such test case in the old integration test suite.

Reply:
K. Let's roll with it :+1: